<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zweitegrund_gui</class>
 <widget class="QDialog" name="zweitegrund_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2. &quot;geod√§tische Hauptaufgabe&quot;</string>
  </property>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>69</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="x1">
      <property name="text">
       <string>x&lt;sub&gt;1&lt;/sub&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_x1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>40</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="y1">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;y&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_y1"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_gauss">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>160</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="umrechnen">
      <property name="text">
       <string>berechnen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Testwerte">
      <property name="text">
       <string>Testwerte</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>100</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="y1_2">
      <property name="text">
       <string>y&lt;sub&gt;2&lt;/sub&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_y2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>129</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="x2">
      <property name="text">
       <string>x&lt;sub&gt;2&lt;/sub&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_x2"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_s12">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>230</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Rechtswert_2">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>78</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Strecke s&lt;span style=&quot; vertical-align:sub;&quot;&gt;1, 2         &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Rechtswert">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>200</y>
     <width>136</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Richtungswinkel t&lt;sub&gt;1, 2&lt;/sub&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_t12">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
